integration/querier_test.go

```diff
@@ -85,7 +85,8 @@ func testQuerierWithBlocksStorageRunningInMicroservicesMode(t *testing.T, series
 		"-blocks-storage.tsdb.retention-period":    "1ms", // Retention period counts from the moment the block was uploaded to storage so we're setting it deliberatelly small so block gets deleted as soon as possible
 
 		// Enable protobuf format so that we can use native histograms.
-		"-query-frontend.query-result-response-format": "protobuf",
+		"-query-frontend.query-result-response-format":  "protobuf",
+		"-ingester.native-histograms-ingestion-enabled": "true",
```

Comment:
Nit: it's not clear why this is here, perhaps a comment would be good idea?

Reply:
Actually I think we don't need this at all as the e2e ingester service on the sparsehistogram branch already enables native histograms: https://github.com/grafana/mimir/blob/543ced934d772d57d5b1edf42043a2ad9b6b9629/integration/e2emimir/services.go#L146

I've tested it and also tested explicitly disabling native histograms on this line, which resulted in failed test, so the common service definition is definitely in effect.

Reply:
oh, this was PR for `sparesehistogram` branch... my bad :(